### ISSUE TITLE
Resource Usages context menu and panel doesn't show resources accessed via ResourceDescriptorHeap indexing (bindless)

### ISSUE THREAD

**Issue by kylawl** [baldurk/renderdoc#3278]
## Description

Resource _Usage in frame_ and _Used:_ context menu do not display usages resulting from bindless access.

## Steps to reproduce

1. Select a draw or dispatch which uses ResourceDescriptorHeap indexing
2. Right click on a ResourceDescriptorHeap indexed resource.
3. Note: the _Used_ context menu will only contain usages where the resource was bound and other non-bindless work like barriers
4. Select _Open in Resource Inspector_, the same subset of usages as the context menu is only visible there.

## Environment

* RenderDoc version: v1.31
* Operating System: Windows 10.0.19045 Build 19045
* Graphics API: D3D12

Thanks Baldur!

**Comment by baldurk** [comment]
Yes the resource usage is tracked via what bindings are used on the CPU, so in the case of bindless access (which means mostly any access via a large arrayed binding in the shader) rather than reporting every binding as used everywhere, they are reported as used nowhere.

It's not clear if it's technically feasible to track accesses that are only possible to account for on the GPU.
